packages/@aws-cdk/aws-appmesh/lib/route-spec.ts

```diff
@@ -68,6 +69,76 @@ export interface HttpRouteSpecOptions {
    * @default - None
    */
   readonly timeout?: HttpTimeout;
+
+  /**
+   * The retry policy
+   * @default - no retry policy
+   */
+  readonly retryPolicy?: HttpRetryPolicy;
+}
+
+/**
+ * HTTP retry policy
+ */
+export interface HttpRetryPolicy {
+  /**
+   * Specify HTTP events on which to retry
+   * @default - no retries for http events
+   */
+  readonly httpRetryEvents?: HttpRetryEvent[];
+
+  /**
+   * The maximum number of retry attempts
+   */
+  readonly maxRetries: number;
+
+  /**
+   * The timeout for each retry attempt
+   */
+  readonly perRetryTimeout: cdk.Duration;
```

Comment:
How about just calling this `retryTimeout`?

Reply:
I'd like that, but I do have a nit. The CDK [design guidelines](https://github.com/aws/aws-cdk/blob/master/DESIGN_GUIDELINES.md#naming) want us to use the terminology of the original service documentation. The CloudFormation calls it `PerRetryTimeout` ([gRPC](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-appmesh-route-grpcretrypolicy.html#cfn-appmesh-route-grpcretrypolicy-perretrytimeout), [HTTP](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-appmesh-route-httpretrypolicy.html#cfn-appmesh-route-httpretrypolicy-perretrytimeout)). What do you think given this?

Reply:
I think `retryTimeout` is clear enough to not cause any confusion here.